index.js

```diff
@@ -27,6 +27,24 @@ module.exports = {
                 base = base + "/";
             }
 
+            // replace placeholder with actual branch name
+            // if Git is installed
+
+            // Match a branch name in brackets
+            base = base.replace(/\[.+\]/, function gitBranch(defaultBranch) {
+              var sys = require('sys')
+              var spawn = require('child_process').spawnSync;
+              // Ask git for the current branch name
+              var run_cl = spawn("git", ["rev-parse", "--abbrev-ref", "HEAD"]);
```

Comment:
I think this can be done using npm package - https://github.com/nodegit/nodegit

Can you please check this? Thanks for PR. :-)